Assets/Shopify/UIToolkit/Editor/ShopCredentialsVerifier.cs

```diff
@@ -34,12 +34,11 @@ public class ShopCredentialsView : IShopCredentialsView {
             DrawMessageBox();
             DrawActionButton();
             EditorGUILayout.Separator();
-            serializedObject.ApplyModifiedProperties();
```

Comment:
Is this intentional?

Reply:
yea because it should be called by the "user" of this class instead of here.... this thing just makes changes